### DISCUSSION TITLE
Mapping Cargo internal directory to external directory fails

### DISCUSSION THREAD

**Discussion by SamTV12345** [BlackDex/rust-musl#2] [Q&A]
I have the problem that when I map the internal cargo directory to the external one that it cannot get a lock to the Cargo directory. Mapping that to an existing directory would speed up the pipeline tremendously as it doesn't need to reindex and download the crates everytime.

**Comment by BlackDex** [comment]
Could you provide an example on how you run this?
That would make it easier for me to try and reproduce.
Also what are the rights, user/group of the folder

**Reply by SamTV12345** [reply]
755 on the .cargo directory and it belongs to root root. .package-cache 544, bin 755, env 544, registry 755 . Everything owned by root and group root.
